pkgs/tools/filesystems/bees/default.nix

```diff
@@ -58,7 +55,7 @@ let
 
 in
 
-runCommand "bees-service-${version}" {
```

Comment:
Hm, I wonder what the point of this was. This would explain why it was outside of stdenv.mkDerivation.

Reply:
Yes. I dropped the version here in a separate commit. I'd rather drop it from the PR and keep version and hash outside of stdenv.mkDerivation, but both is perfectly fine for me.